src/Umbraco.Cms.Api.Management/Controllers/AuditLog/CurrentUserAuditLogController.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Api.Common.ViewModels.Pagination;
+using Umbraco.Cms.Api.Management.Factories;
+using Umbraco.Cms.Api.Management.ViewModels.AuditLogs;
+using Umbraco.Cms.Core;
+using Umbraco.Cms.Core.Exceptions;
+using Umbraco.Cms.Core.Models;
+using Umbraco.Cms.Core.Models.Membership;
+using Umbraco.Cms.Core.Security;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.Core.Services.OperationStatus;
+using Umbraco.New.Cms.Core.Models;
+
+namespace Umbraco.Cms.Api.Management.Controllers.AuditLog;
+
+public class CurrentUserAuditLogController : AuditLogControllerBase
+{
+    private readonly IAuditService _auditService;
+    private readonly IAuditLogViewModelFactory _auditLogViewModelFactory;
+    private readonly IBackOfficeSecurityAccessor _backOfficeSecurityAccessor;
+    private readonly IUserService _userService;
+
+    public CurrentUserAuditLogController(
+        IAuditService auditService,
+        IAuditLogViewModelFactory auditLogViewModelFactory,
+        IBackOfficeSecurityAccessor backOfficeSecurityAccessor,
+        IUserService userService)
+    {
+        _auditService = auditService;
+        _auditLogViewModelFactory = auditLogViewModelFactory;
+        _backOfficeSecurityAccessor = backOfficeSecurityAccessor;
+        _userService = userService;
+    }
+
+    [HttpGet]
+    [MapToApiVersion("1.0")]
+    [ProducesResponseType(typeof(PagedViewModel<AuditLogWithUsernameViewModel>), StatusCodes.Status200OK)]
+    public async Task<IActionResult> CurrentUser(Direction orderDirection = Direction.Descending, DateTime? sinceDate = null, int skip = 0, int take = 100)
```

Comment:
Here again I think it would give a lot of value to send out the keys of the entities as well :D